### PR TITLE
Fix yamllint error

### PR DESCRIPTION
**Pull request by prietyc123**
error log: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/48881/rehearse-48881-pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-yamllint/1759469911152791552

### CODE REVIEW

File: new-for-dance/pipelines-helm/pipeline-build.yaml [hunk 1]
```diff
@@ -2,23 +2,23 @@
 apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
-  name: {{ .Values.app.name }}-build
+  name: {{.Values.app.name}}-build
```
[the remark this answers is not in the document]
Comment (prietyc123) [reply new-for-dance/pipelines-helm/pipeline-build.yaml]: Done

**Comment by mmorhun** [comment]
Many definitions are just synced from build-definitions and will be overwritten the next sync.
We have to fix build-difinitions first.

**Comment by prietyc123** [comment]
> Many definitions are just synced from build-definitions and will be overwritten the next sync.
> We have to fix build-difinitions first.

Do you mean build definitions does not check yamllint? If yes then we need to sync that first

**Comment by prietyc123** [comment]
@mmorhun Build definition already has yamllint check https://github.com/redhat-appstudio/build-definitions/blob/c3a5e241eeeaede243e93491674f0237f354d4e4/.tekton/pull-request.yaml#L41

**Pull request closed without merging**